src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -144,42 +148,36 @@ enum SideMenuButtonType {
-        if #available(iOS 11.0, *) {
-            continueButton.topAnchor.constraint(equalTo: self.safeAreaLayoutGuide.topAnchor, constant: marginTopBottom).isActive = true
-        } else {
-            continueButton.topAnchor.constraint(equalTo: self.topAnchor, constant: marginTopBottom).isActive = true
-        }
+        let continueButton = setupButton(imageName: "stage_dialog_button_continue", selector: #selector(delegate?.continueAction), type: .landscapeFirstColumn)
+        continueButton.topAnchor.constraint(equalTo: backLabel.bottomAnchor, constant: marginTopBottom * 2).isActive = true
 
-        let continueLabel = setupLabel(title: kLocalizedContinue, selector: #selector(delegate?.continueAction), type: .landscapeSecondColumn)
-        continueLabel.topAnchor.constraint(equalTo: continueButton.bottomAnchor, constant: marginLabel).isActive = true
+        let continueLabel = setupLabel(title: kLocalizedContinue, selector: #selector(delegate?.continueAction), type: .landscapeFirstColumn)
+        continueLabel.topAnchor.constraint(equalTo: continueButton.bottomAnchor).isActive = true
+        continueLabel.bottomAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
 
         let restartButton = setupButton(imageName: "stage_dialog_button_restart", selector: #selector(delegate?.restartAction), type: .landscapeFirstColumn)
-        restartButton.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: (marginTopBottom - marginLabel) * -1).isActive = true
+        restartButton.topAnchor.constraint(equalTo: continueLabel.bottomAnchor, constant: marginTopBottom * 2).isActive = true
 
         let restartLabel = setupLabel(title: kLocalizedRestart, selector: #selector(delegate?.restartAction), type: .landscapeFirstColumn)
         restartLabel.topAnchor.constraint(equalTo: restartButton.bottomAnchor, constant: marginLabel).isActive = true
+        restartLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: marginTopBottom  * -5).isActive = true
 
         let screenshotButton = setupButton(imageName: "stage_dialog_button_screenshot", selector: #selector(delegate?.takeScreenshotAction), type: .landscapeSecondColumn)
-        screenshotButton.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: (marginTopBottom - marginLabel) * -1).isActive = true
-
-        let screenshotLabel = setupLabel(title: kLocalizedPreview, selector: #selector(delegate?.takeScreenshotAction), type: .landscapeSecondColumn)
-        screenshotLabel.topAnchor.constraint(equalTo: screenshotButton.bottomAnchor, constant: marginLabel).isActive = true
+        screenshotButton.topAnchor.constraint(equalTo: restartLabel.bottomAnchor, constant: marginLabel * 5).isActive = true
+        screenshotButton.leftAnchor.constraint(equalTo: self.leftAnchor, constant: marginTopBottom * 1).isActive = true
+        screenshotButton.rightAnchor.constraint(equalTo: self.centerXAnchor, constant: marginLabel).isActive = true
```

Comment:
please set the right anchor to the leftAnchor of the Axes Button, otherwise the clickable box is way to big and overlaps with the AxesButton